apm-sniffer/apm-sdk-plugin/mysql-5.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/jdbc/mysql/v5/ConnectionCreate5xInterceptor.java

```diff
@@ -42,7 +43,8 @@ public void beforeMethod(Class clazz, Method method, Object[] allArguments, Clas
     public Object afterMethod(Class clazz, Method method, Object[] allArguments, Class<?>[] parameterTypes,
         Object ret) {
         if (ret instanceof EnhancedInstance) {
-            ConnectionInfo connectionInfo = ConnectionCache.get(allArguments[0].toString(), allArguments[1].toString(), allArguments[3].toString());
+            String database = Objects.isNull(allArguments[3]) ? "" : allArguments[3].toString();
```

Comment:
URL could be NULL? Because? Users forgot to set?

Reply:
jdbc url like 
jdbc:mysql://ip:port?zeroDateTimeBehavior=convertToNull&useUnicode=true&characterEncoding=UTF8&connectTimeout=10000&socketTimeout=20000&useSSL=false&useTimezone=true&serverTimezone=GMT%2B8

the database not assign in jdbcurl, this can get null databse